app/tests/data_backend_test/test_spark.py

```diff
@@ -0,0 +1,9 @@
+# go three folders up
+from ...data_backend import spark_oceandb as sp
+
+
+
+
+spark = sp.create_spark_session()
+print('Session Started')
```

Comment:
Please do not use prints in the codebase. Maybe if you need to display any feedback from the app then it's better to use some logger (loguru) with info(). Just for the sake of keeping console output clean.

Reply:
changed